django/applications/catmaid/control/user_evaluation.py

```diff
@@ -30,12 +30,12 @@ def _find_nearest(tree, nodes, loc1):
             min_sqdist = dsq
             closest = node
 
-    return node, min_sqdist
+    return closest, min_sqdist
```

Comment:
I think this was a bug. Pretty sure this is what this function meant to do.

Reply:
Yes, good catch!